packages/shared/src/components/fields/Checkbox.tsx

```diff
@@ -18,6 +18,7 @@ export interface CheckboxProps extends InputHTMLAttributes<HTMLInputElement> {
   id?: string;
   children?: ReactNode;
   className?: string;
+  checkmarkClassName?: string;
```

Comment:
Decide to add secondary as it has too many use-cases already and would be a big change